src/status/libstatus/accounts.nim

```diff
@@ -238,6 +238,13 @@ proc verifyAccountPassword*(address: string, password: string): bool =
 
   return false
 
+proc changeDatabasePassword*(keyUID: string, password: string, newPassword: string): bool =
+  let hashedPassword = hashPassword(password)
+  let hashedNewPassword = hashPassword(newPassword)
+  let changeResult = $status_go.changeDatabasePassword(keyUID, hashedPassword, hashedNewPassword)
```

Comment:
Should this call be wrap in a try/except with a return false in case of error and a debug with the error message?

Reply:
Good idea, will do!